build-setup-nolog.sh

```diff
@@ -102,6 +91,16 @@ else
     target_chipyard_dir=$RDIR/target-design/chipyard
 fi
 
+# Enable latest Developer Toolset for GNU make 4.x
+devtoolset=''
+for dir in /opt/rh/devtoolset-* ; do
```

Comment:
`scripts/build-toolchains.sh` in Chipyard now explicitly checks for the make version, so sourcing devtoolset can be made optional (to accommodate non-CentOS hosts).  Having an obsolete make version will still be caught early in the build process.